docs/how-to-guides/debug-autoware.md

```diff
@@ -63,6 +63,52 @@ ros2 run rqt_console rqt_console
    ros2 run rqt_plot rqt_plot
    ```
 
+### Using ros2_graph
+
+[`ros2_graph`](https://github.com/kiwicampus/ros2_graph) can be used to generate [mermaid](https://mermaid.js.org/#/) description of ROS2 graphs to add on your markdown files.
```

Comment:
We need a space.
https://discourse.ros.org/t/should-i-put-a-space-between-ros-and-2/27949

```suggestion
[`ros2_graph`](https://github.com/kiwicampus/ros2_graph) can be used to generate [mermaid](https://mermaid.js.org/#/) description of ROS 2 graphs to add on your markdown files.
```